### PR TITLE
TP-616: Configure ELK logging, Sentry and Pingdom

### PR DESCRIPTION
**Pull request by NDevox**

### CODE REVIEW

File: common/views.py [hunk 1]
```diff
@@ -29,6 +30,18 @@ def index(request):
...
+        "<pingdom_http_custom_check><status><strong>OK</strong></status>"
+        "<response_time><strong>1</strong></response_time>"
+        "</pingdom_http_custom_check>",
+        content_type="text/xml",
+    )
+    response["Cache-Control"] = "no-cache, no-store, must-revalidate"
+
+    return response
+
+
```
Comment (simonwo) [inline common/views.py:44]: I beleive that now we have attached services (postgres, redis) the healthcheck is also meant to check that they are accessible?

At which point the response time maybe also needs to reflect truth.

**Review by simonwo — CHANGES_REQUESTED** [review]

File: common/views.py (hunk 1 above)
Comment (NDevox) [reply common/views.py:44]: Fun when you go looking for examples of people doing this and find examples such as https://github.com/uktrade/data-workspace/blob/97bd538617c4835a749fe166eee7e4fca0dd6acc/dataworkspace/dataworkspace/apps/core/views.py#L41

Comment (stuaxo) [reply common/views.py:44]: I guess this should try and connect to celery too.

Could be a nice excuse to use async to start all checks the subtasks then gather the results.

**Review by simonwo — APPROVED** [review]

File: common/views.py (hunk 1 above)
Comment (NDevox) [reply common/views.py]: This is done now @simonwo

Comment (NDevox) [reply common/views.py]: Ha yeah not sure I'd bother with the async stuff - don't think the return is great enough - given that it's only pingdom.

Testing for celery is an interesting idea. I don't know how much I want to lean into our reliance on it before we're properly using it just yet. But once it becomes integral we should definitely put it on the list.

**Pull request merged**